lib/matplotlib/tests/test_ticker.py

```diff
@@ -521,6 +525,13 @@ def test_sublabel(self):
         ax.set_xlim(0.5, 0.9)
         self._sub_labels(ax.xaxis, subs=np.arange(2, 10, dtype=int))
 
+    @pytest.mark.parametrize('val', call_data)
```

Comment:
If you're not going to provide multiple values for `val`, I wouldn't bother parametrizing the test. Just define `val`/`call_data` inside the test.